Source/WebGPU/WebGPU/Device.mm

```diff
@@ -309,17 +309,15 @@ static uint32_t computeAppleGPUFamily(id<MTLDevice> device)
 {
 #if ENABLE(WEBGPU_SWIFT)
     NSError *error = nil;
-    static std::once_flag onceFlag;
```

Comment:
`m_nopVertexFunction` is a member variable, so `call_once` is not going to work since each new call to `requestDevice` creates a new Device.

Alternative is to make `m_nopVertexFunction` a static variable